test/pylib/report_plugin.py

```diff
@@ -12,21 +12,21 @@
 
 class ReportPlugin:
     config = None
-    mode = None
+    build_mode = None
```

Comment:
When I asked to rename the fixture "mode", I didn't mean you need to rename all the local variables named "mode". But Ok :-)